Post 41 docs fixes (#108)
Ultramarine-Linux/wiki: src/content/docs/en/anywhere/wsl.mdx

## System Requirements

<Alert type="warning">

>     Ultramarine's WSL application is still is currently in review by the Microsoft Store team. You will not be able to install Ultramarine WSL with the below instructions until the application is aproved for the store. When Ultramarine WSL is live, this message will be removed.

Reviewer: ```suggestion
    Ultramarine's WSL application is still is currently in review by the Microsoft Store team. You will not be able to install Ultramarine WSL with the below instructions until the application is approved for the store. Check back soon!
```
